charters/IngestSupport/charter.md

```diff
@@ -0,0 +1,100 @@
+# Ingest Support
```

Comment:
_Ingest Assist[ance] (?)_ to avoid _Ingest Support **supports**_ ... or _Ingest Central_ which I've loved every since I discovered the repo.

Reply:
I am going to change the usage of support as a verb to assist as I prefer Support as the group name it being an actual noun